source/text/Format.cpp

```diff
@@ -89,6 +89,32 @@ string Format::Credits(int64_t value)
 
 
 
+// Convert a time in seconds to years/days/hours/minutes/seconds
+std::string Format::PlayTime(int time)
```

Comment:
We probably should use a different parameter name, given that elsewhere we're using `time` to refer to a function.

Reply:
Would playTime work, if that's also the name of the variable being (indirectly) passed? Trying to think of alternatives that aren't too wordy.

Reply:
Yes, that would work, or e.g. `timeVal`. My concern was that here we are declaring a function parameter named `time` of type `int`, while in other code we are using a variable named `time` that is a function with a particular signature.

The names can be equivalent with without technical issue (the compiler can deduce what `time` should mean in both cases), but my concern was for the sake of a user that is looking at the code.